cli/integrity/integrity.go

```diff
@@ -6,6 +6,8 @@ import (
 	"github.com/spf13/pflag"
 )
 
+var FileRepository Repository = dummyRepository{}
```

Comment:
A global object is not a good idea. It's hard to test, it's hard to maintain.

It's much better if it's tied to some common command context. Is it too hard to implement?

Reply:
I don't think it would be _prohibitively_ hard, but there are certainly places where the command context is not passed, but we would like to use the integrity module.

Reply:
What places?

Reply:
> In case of script/cluster instances, this is also not immediately obvious, although I believe we can store a pointer to a repository inside the instance structure.

What the case?